Automattic/wp-calypso
# Add migration started step

Related to #91852

## Proposed Changes

* Add the content and styles to the Migration Started step.

## Testing Instructions



* Make sure you have the `migration-flow/new-migration-instructions-step` feature flag enabled.
* Create a new site with the Import intent.
* Follow all the steps until the Migration Started step.
* Check that it works well in different screen sizes.

## Screenshots

<img width="1270" alt="Screenshot 2024-07-16 at 15 19 30" src="https://github.com/user-attachments/assets/9660bffb-9e1f-41cb-81e2-19620b290254">

<img width="398" alt="Screenshot 2024-07-16 at 15 19 22" src="https://github.com/user-attachments/assets/8897c670-307e-4377-831e-08b4eec7c96e">

cc @fditrapani to validate the responsive design for small screens, where I hid the image to have enough space for the button.

## Pre-merge Checklist
